cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectConfiguration.java

```diff
@@ -34,13 +34,25 @@ public class KafkaConnectConfiguration extends AbstractConfiguration {
     }
 
     /**
-     * Constructor used to instantiate this class from JsonObject. Should be used to create configuration from
-     * ConfigMap / CRD.
+     * Copy constructor which creates new instance of the Kafka Connect configuration from existing configuration.
+     * It is useful when you need to modify an instance of the configuration without permanently changing the original.
+     *
+     * @param configuration User provided Kafka Connect configuration
+     *
+     */
+    public KafkaConnectConfiguration(KafkaConnectConfiguration configuration) {
+        super(configuration);
+    }
+
+
+    /**
+     * Copy constructor which creates new instance of the Kafka Connect configuration from existing configuration.
+     * It is useful when you need to modify an instance of the configuration without permanently changing the original.
      *
      * @param reconciliation  The reconciliation
      * @param jsonOptions     Json object with configuration options as key ad value pairs.
      */
-    public KafkaConnectConfiguration(Reconciliation reconciliation, Iterable<Map.Entry<String, Object>> jsonOptions) {
+    public KafkaConnectConfiguration(Iterable<Map.Entry<String, Object>> jsonOptions, Reconciliation reconciliation) {
```

Comment:
Can we have `reconciliation` as first parameter like in `KafkaMirrorMaker2Configuration` and `KafkaConfiguration`?